src/js/stores/MarathonStore.js

```diff
@@ -159,6 +159,27 @@ var MarathonStore = Store.createStore({
     return this.get('apps')[name];
   },
 
+  getServiceFromTaskID: function (taskID) {
+    let serviceName = taskID.split('.')[0].split('_');
+    serviceName = serviceName[serviceName.length - 1];
```

Comment:
This doesn't seem very robust. I'd like to suggest extracting the `taskID` parsing to a Util so that we can test the hell out of it and generally to add a few somewhat more defensive checks.